### PR TITLE
Add & remove

### PR DESCRIPTION
**Pull request by JuanLPalacios**
implementation of the CRUD methods.

### CODE REVIEW

File: src/add-and-remove.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const TODO = 'todo';
```
Comment (rustam-tolipov) [inline src/add-and-remove.js:1]: ## GOOD JOB SO FAR 👏

- [ ] There are parts that are missing in the `readme` file. Please kindly add the screenshot of the app and the MIT license ✅

File: src/index.js [hunk 2]
```diff
@@ -1,32 +1,40 @@
...
+    li.querySelector('.delete').addEventListener('mousedown', () => {
+      remove(element, todo);
+      update();
+    });
+    li.querySelector(`#txt-${i}`).addEventListener('blur', (e) => {
+      editDescription(element, e.target.value);
+      save(todo);
+      update();
+    });
+  });
```
Comment (rustam-tolipov) [inline src/index.js]: - [ ] When I add a new task to the to-do list, the page is refreshing. Please also fix it by removing it 👌

File: src/index.js [hunk 3]
```diff
@@ -1,32 +1,40 @@
...
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Add and remove items.',
-    completed: false,
-    index: 2,
-  },
-];
+const todo = load() || [];
```
Comment (rustam-tolipov) [inline src/index.js:6]: - [ ] Finally, the index of the elements should start with `1` in todo data, but in yours, it's starting from `0`. You can set the started number to 1 👍

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hi @JuanLPalacios , 

Good job so far!

### HIGHLIGHTS!

- No linters errors.💯
- Descriptive Readme.md. 💯


There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @JuanLPalacios,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**